cogs/staffapps_frontend.py

```diff
@@ -14,6 +14,8 @@
 from util.limiter import limit
 from util.logger import log
 
+STAFF_CHANNEL_ID = int(getenv("STAFF_CHANNEL", -1))
```

Comment:
We should probably add asserts to make sure non-optional environmental variables always exist.
AKA reject bot launches without all proper env variables set.

Reply:
I think that's a good idea, but lets discuss that in another PR. This one is already pretty big

Reply:
Yeah, indeed. I'll make an issue for this